scripts/deploy.sh

```diff
@@ -27,6 +27,12 @@ function run_ansible {
   openstack-ansible ${ANSIBLE_PARAMETERS} --forks ${FORKS} $@
 }
 
+apt-get install python-dev -y
+curl https://raw.githubusercontent.com/pypa/pip/7.1.2/contrib/get-pip.py -o /opt/get-pip.py
```

Comment:
So, does the recent changes around 8.0.2 fix that? I swear there was hubub the day of 8.0.0 that people needed to specify a version.

https://raw.githubusercontent.com/pypa/pip/8.0.2/contrib/get-pip.py

now ponts to

https://bootstrap.pypa.io/get-pip.py

which has:

```
   # We want to support people passing things like 'pip<8' to get-pip.py which
    # will let them install a specific version. However because of the dreaded
    # DoubleRequirement error if any of the args look like they might be a
    # specific for one of our packages, then we'll turn off the implicit
    # install of them.
```